content/chat/moderation/custom/index.textile

```diff
@@ -0,0 +1,84 @@
+---
+title: Custom Moderation
+meta_description: "Detect and remove unwanted content in a Chat Room using a custom provider"
+product: chat
+---
+
+There may be situations where you have trained your own model, or you want to apply proprietary logic using your own infrastructure, whilst performing moderation.
+
+Ably provides simple APIs to allow your moderation logic to prevent harmful content from being present in your chat room.
+
+h2(#beforepublish). Before publish
+
+Before publish moderation is where your moderation logic is invoked before the message is published to your chat room. This has the benefit of preventing harmful content from ever entering your chat room, at the cost of some latency in invoking your moderation logic as part of the publish path.
+
+h3(#configuration). Configuration
```

Comment:
This is called 'rules fields' in the Hive ones - don't mind which we go for but they should be consistent. 

We should also mention where you use the fields (i.e. dashboard and Control API).

Reply:
acae84df9